swagger.yml

```diff
@@ -87,27 +87,23 @@ paths:
       produces:
       - "application/json"
       parameters:
-      - name: "username"
-        in: "query"
-        description: "The user name for login"
-        required: true
-        type: "string"
-      - name: "password"
-        in: "query"
-        description: "The password for login in clear text"
-        required: true
-        type: "string"
+       - in: "body"
+         name: "body"
+         description: "user details parameters"
+         required: true
+         schema:
+          $ref: "#/definitions/Register"
       responses:
         "200":
           description: "successful operation"
           schema:
             $ref: "#/definitions/Register"
         "401":
-          $ref: '#/responses/UnauthenticatedError'
+           description:  "Authentication information is missing or invalid"
         "403":
-          $ref: '#/responses/UnauthorizedError'
+           description:  "User is not authorised to make the request"
```

Comment:
Please use the parameterised values here, there is no point repeating the messages again. The last block has the common 401, 403 and 503 parameterised